Pseudocode/Greedy Algorithms/Activity Selection/SourceCode.tex

```diff
@@ -0,0 +1,52 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Remove the 'end' at the end of the algorithm
+\usepackage[noend]{algpseudocode}
+
+% Define Left Justified Comments
+\algnewcommand{\LeftComment}[1]{\Statex \(\triangleright\) #1}
+
+% Remove the Numbering of the Algorithm
+\usepackage{caption}
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+\begin{document}
+
+\begin{algorithm}
+
+  \caption{Maximize the number of activities without overlapping schedules}
+  \begin{algorithmic}[1]
+    \Ensure One Based Indexing for the Array
+    \Statex
```

Comment:
Do you mean a blank line before this `\Statex` and after `\Ensure`

Reply:
No, a blank line after the left comment

Reply:
There is already a blank line here